Jenkinsfile

```diff
@@ -318,7 +319,14 @@ pipeline {
 
                         stash name:'mac-exe', includes:'bin/*'
                     }
-                    post { always { runShell("rm -rf ./*") }}
+                    post {
+                        always {
+                            runShell("""
+                            opam switch 4.07.0
```

Comment:
So this is a temporary artifact of the build process. Our Mac machines aren't containerized, and the `opam switch` command is side-effecting. If we didn't switch _back_, temporarily, it would fail to compile `master` on Mac (or any branch which still uses 4.07)

After we merge this, that line will go away

Reply:
Ah, so the Mac was updated to 4.12?

Reply:
Yes we've updated it earlier today and running both 4.12 and 4.07